lib/traits/slim/device_slim_widget.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/material.dart';
```

Comment:
Is this UnknownDevice?

Reply:
This is more of a generic base so whenever we need something new it's just plug and play whatever icons and content.

Reply:
Ok, I see. I think this maybe one of those instances where it makes intention clearer to name this as a baseDevice.